lib/rubocop/cop/layout/space_around_operators.rb

```diff
@@ -50,6 +50,20 @@ module Layout
       #
       #   # good
       #   a ** b
+      #
+      # @example EnforcedStyleForSlashInRationalLiterals: no_space (default)
```

Comment:
Probably we can we should shorten this to `EnforcedStyleForRationalLiterals`

Reply:
(I mean it's clear that something like this must apply for the slash, as it can't be anything else)